addOns/ascanrulesBeta/src/test/java/org/zaproxy/zap/extension/ascanrulesBeta/HiddenFilesScanRuleUnitTest.java

```diff
@@ -478,6 +478,140 @@ void shouldRaiseAlertIfTestedUrlRespondsOkWithRelevantContentAndAppropriateNotCo
         assertEquals(rule.getReference(), alert.getReference());
     }
 
+    @Test
+    void
+    shouldRaiseAlertIfTestedUrlRespondsOkWithRelevantContentSVNEntries()
```

Comment:
I think these can be put on a single line, the one before is really long hence it being wrapped.